docs/examples/PanelCollapsible.js

```diff
@@ -12,7 +12,11 @@ class Example extends React.Component {
         <Button onClick={ ()=> this.setState({ open: !this.state.open })}>
           click
         </Button>
-        <Panel collapsible expanded={this.state.open}>
+        <Panel
+          collapsible
+          id="collapsible-panel-example"
+          expanded={this.state.open}
+        >
           Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
```

Comment:
Well I'd do something like

```js
<Panel>
  <Panel.Body collapsible>
    foo
  </Panel.Body>
</Panel>
```

or

```js
<Panel>
  <Panel.Collapse>
    <Panel.Body>
      foo
    </Panel.Body>
    <Table />
    <Panel.Body>
      bar
    </Panel.Body>
  </Panel.Collapse>
</Panel>
```

It's maybe a little counterintuitive that you can't use two `<Panel.Body collapsible>`, but I don't think it's be the worst thing ever.